### PR TITLE
Polish recipe generation on gemini, attempt on chatterbot, add tests

### PR DESCRIPTION
**Pull request by will-hoover**
Chatterbot:
- [x] add list of questions and resulting recipes
- [x] ensure that chatterbot Ramsay generates the given recipes

Gemini:
- [x] ensure the Ramsay AI outputs correct recipes
- [x] ensure the recipes are properly formatted

Both
- [x] write tests that pass on CI and can be repeated in the future

### CODE REVIEW

**Comment by will-hoover** [comment]
The current number of recipes has been deemed ok due to work happening to seamlessly import recipe responses from gemini.

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ backend/.env
 backend/.python-version
 backend/__pycache__/*
 backend/db.sqlite3*
+backend/tests/__pycache__/*
```
Comment (brandonmontijo) [inline .gitignore]:
```suggestion
backend/tests/__pycache__/*

```

**Review by brandonmontijo — APPROVED** [review]
LGTM - if the commented code in `backend/tests/test_gemini.py` passes, I think it would be worth uncommenting

**Comment by will-hoover** [comment]
> LGTM - if the commented code in `backend/tests/test_gemini.py` passes, I think it would be worth uncommenting

I did just do that locally, I'll push that and ensure it does before I push this.

**Comment by ATY02** [comment]
For me, if the inputs are not exactly the loaded ones, it will not give the correct answer. Ex: if I ask "Can you give me a recipe for pancakes?" it will not give me the pancakes recipe.

**Pull request merged**